### DISCUSSION TITLE
Secure Spreadsheets on iOS is basically unusable

### DISCUSSION THREAD

**Discussion by stkr22** [standardnotes/forum#2914] [General]
**Describe the bug**
The header and the keyboard are so close together it is basically impossible to insert data or even see the data you already inserted.

**To Reproduce**
1. Create new Secure Spreadsheet
2. Click on cell

**Expected behavior**
Dunno maybe fix the header with menus to the top so it does not follow the user when scrolling or allow the user to hide the header.

**Screenshots**
![image](https://user-images.githubusercontent.com/98834911/205242126-807b1a40-991e-4159-b153-ffb1384a3d82.jpeg)

**Smartphone (please complete the following information):**
 - Device: iPhone SE
 - OS: iOS 16.1.1
 - Version 3.114.0

**Comment by effieeee** [comment]
Hi @stkr22, if I'm not mistaken, the fixed position of the note title and options header isn't something that can be changed for a specific note type at the moment, but we'll see if it can be made an option

**Comment by stkr22** [comment]
Hi @effieeee, 
Thanks for the quick response. 

Yeah sure, it was just a suggestion how it might be solved. Currently it cannot be used because keyboard and menu(header) are blocking the full view together. That means the editor is basically broken and it used to work.

It could also help to not show the footer here(which does not make sense anyway): 

![image](https://user-images.githubusercontent.com/98834911/205429293-bc92a892-6b5d-4d07-9c73-f25a57879242.jpeg)

**Comment by stkr22** [comment]
Hi @effieeee, any updates on this?

**Comment by effieeee** [comment]
None so far I’m afraid, current efforts have been on other persisting issues, and developments with the Super note type 😞

**Comment by stkr22** [comment]
Hi @effieeee,
Thanks again for the quick feedback. 

I think the change that actually broke the note type for me is that the footer used to be fixed to the bottom and is now scrolling with the user window once you enter edit mode. This is not a feature request it is a bug that was introduced.

Anyway, your response feels a little off. Again I'm a **paying customer** and the securesheets used to work just fine until a couple of month ago. Since then I cannot use the note type anymore on mobile. My expectation was that this is a mid-high priority bug that should be fixed.

**Comment by myreli** [comment]
One idea that comes to mind is applying the same behaviour to the menu with Super (hiding the buttons unless the user clicks the note's title).
